content/client-server-api/_index.md

```diff
@@ -2540,6 +2540,9 @@ return a standard error response of the form:
 The `retry_after_ms` key SHOULD be included to tell the client how long
 they have to wait in milliseconds before they can try again.
 
+The `Retry-After` HTTP header MAY be specified by the server. When this header
```

Comment:
It might be nice to link to the specification for this header?

```suggestion
The [`Retry-After`](https://www.rfc-editor.org/rfc/rfc9110#field.retry-after) HTTP header MAY be specified by the server. When this header
```